keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/VerificationRepositoryTests.kt

```diff
@@ -158,17 +158,17 @@ abstract class VerificationRepositoryTests<IMPLEMENTATION : VerificationReposito
 
     context.setup()
 
-    subject.updateState(context, verification, RUNNING)
+    subject.updateState(context, verification, PENDING)
```

Comment:
Hmm, while I like the idea of DRYing things up, I think `RUNNING` and `PENDING` mean two different things.

Reply:
That would mean changing existing constraints to also use that new status consistently